### ISSUE TITLE
Remove feedback pagefrom the task-switching task

### ISSUE THREAD

**Issue by arielle68** [EmotionCognitionLab/pvs#50]
Please remove the feedback page from the single blocks and mixed block of task-switching task. So after pressing the button, the cross appears and then the rule for the next trial will appear. Please keep the feedback for the exercise block of the mixed task. 
Thank you.

**Comment by noah10** [comment]
I assume that you want to keep the "Answer faster next time" feedback for everything? Or should we get rid of "Answer faster next time" (for everything except the exercise block) as well?

**Comment by arielle68** [comment]
Yes please  keep the "answer faster next time" feedback for everything.

**Comment by noah10** [comment]
Something I just realized: Removing the feedback means (at least when people answer quickly enough) that we effectively have a 700ms fixation between trials: You give your answer, you get a 500ms post-trial fixation point and then you get a 200ms pre-trial fixation point for the next trial. Is that what you want? (Of course if they don't answer quickly enough they'll get a "Answer faster" message in between those two fixation points.)

**Comment by noah10** [comment]
When you write "So after pressing the button, the cross appears and then the rule for the next trial will appear," did you want the cross to appear for 500ms, 200ms or 700ms? (Or some other amount?) If we just remove the feedback it would appear for 700ms - 500ms for the post-answer fixation, followed immediately by 200ms for the pre-rule fixation for the next trial.

**Comment by arielle68** [comment]
I think 500ms sounds like a more reasonable duration. So we can remove the pre-rule fixation and just keep the post-answer fixation.

**Comment by noah10** [comment]
This has now been fixed - please test it and close the issue if it is working as requested.

**Comment by arielle68** [comment]
Thanks Noah, the task is working fine. Just one minor note, would it be possible to show the cross after the "response faster next time" and also after the feedback part for exercise block? Thanks

**Comment by noah10** [comment]
OK - the cross should now show after feedback (whether "respond faster", incorrect or correct). Please test and close this issue if everything looks like all the changes requested here are working.
